packages/web/components/CurateSpecialties/CurateSpecialties.js

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { useTranslation } from 'react-i18next';
+import { useModal } from '../../hooks';
+
+const CurateSpecialties = ({ data = [] }) => {
+	const { t } = useTranslation(['profile']);
+	const { openModal } = useModal();
+
+	const getChildrenCategory = (category) => {
+		const filteredCategory = data.filter((c) => c.parent_id === category);
+		return filteredCategory.length > 0 ? filteredCategory[0].term : 'Todas';
+	};
+
+	return (
+		<Container>
+			<h3>{t('profile:curateMySpecialties')}</h3>
+			<List>
+				{data.length === 0 ? (
+					<p>Nenhuma área de atuação!</p>
+				) : (
+					data
+						.filter((c) => c.parent_id === null)
+						.map((category) => (
+							<ListItem key={`category_${category.id}`}>
+								<section>
+									<b>{category.term}</b> - {getChildrenCategory(category.id)}
+								</section>
+								<button
+									type="button"
+									onClick={() =>
+										openModal('curateSpecialtiesDelete', {
+											speciality: [
+												category.id,
+												data.filter((c) => c.parent_id === category.id)[0]
```

Comment:
```suggestion
												data.filter((c) => c.parent_id === category.id)[0]
```

Mova isso para uma variável mais acima e utilize um find, é melhor

Reply:
Feito!